docs/samples/Microsoft.ML.Samples/Dynamic/ModelOperations/OnnxConversion.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using Microsoft.ML;
+
+namespace Samples.Dynamic.ModelOperations
+{
+    public static class OnnxConversion
+    {
+        class PlattModelInput
+        {
+            public bool Label { get; set; }
+            public float Score { get; set; }
+        }
+
+        static IEnumerable<PlattModelInput> PlattGetData()
+        {
+            for (int i = 0; i < 100; i++)
+            {
+                yield return new PlattModelInput { Score = i, Label = i % 2 == 0 };
+            }
+        }
+        public static void Example()
+        {
+            
+            var mlContext = new MLContext(seed: 0);
+
+            //Get input data
+            IDataView data = mlContext.Data.LoadFromEnumerable(PlattGetData());
+
+            //Get estimator
+            var calibratorEstimator = mlContext.BinaryClassification.Calibrators.Platt();
+
```

Comment:
Can you please use a pipeline that is similar to something that is already in the other samples for BinaryClassification?